File: java/210525_class/src/com/cm/carApp/customer/Customer.java
package com.cm.carApp.customer;

import com.cm.carApp.car.Car;

public class Customer {
	private String name;
	private String tel;
	private String address;
	private Car car;
	
	public Customer(String name, String tel, String address, Car car) {
		super();
		this.name = name;
		this.tel = tel;
		this.address = address;
		this.car = car;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getTel() {
		return tel;
	}

	public void setTel(String tel) {
		this.tel = tel;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}
	
	public Car getCar() {
		return car;
	}

	public void setCar(Car car) {
		this.car = car;
	}

	public void printCustomerInfo() {
		
		System.out.println("---------------------");
		System.out.println("차량 구매 고객 정보");
		System.out.println("---------------------");
		car.printCarInfo();
		System.out.println("이름: " + name);
		System.out.println("전화: " + tel);
		System.out.println("주소: " + address);
		System.out.println();
	}

}

File: C#/210610/210610_class_car/210610_class_car/model/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _210610_class_car.carApp.model
{
    class Customer
    {
        private string name;
        private string tel;
        private string address;
        private Car car;
       
        public Customer(string name, string tel, string address, Car car)
        {
            this.name = name;
            this.tel = tel;
            this.address = address;
            this.car = car;
        }

        public string Name { get => name; set => name = value; }
        public string Tel { get => tel; set => tel = value; }
        public string Address { get => address; set => address = value; }
        internal Car Car { get => car; set => car = value; }


 
        public void printCustomerInfo()
        {
            Console.WriteLine("이름: " + name);
            Console.WriteLine("연락처: " + tel);
            Console.WriteLine("주소: " + address);
            Car.printCarInfo();
            Console.WriteLine("===========================================\n");
        }
    }
}
